.circleci/config.yml

```diff
@@ -276,15 +343,18 @@ workflows:
                            - master
                            - release-*
                            - rc
-            - unit_tests:
-                requires:
-                    - install
-            - end_to_end_tests:
-                requires:
-                    - install
-            - end_to_end_tests_localdb:
+#            - unit_tests:
+#                requires:
+#                    - install
+#            - end_to_end_tests:
+#                requires:
+#                    - install
+            - end_to_end_tests_IE11:
                 requires:
                     - install
+#            - end_to_end_tests_localdb:
+#                requires:
+#                    - install
```

Comment:
i guess all these ☝️ are temp disabled, but let's not forget to re-enable before merging

Reply:
yup